modules/cli/src/main/scala/scala/cli/commands/Compile.scala

```diff
@@ -77,7 +78,8 @@ object Compile extends ScalaCommand[CompileOptions] with BuildCommandHelpers {
     val threads      = BuildThreads.create()
 
     val compilerMaker = options.shared.compilerMaker(threads).orExit(logger)
-    val configDb = ConfigDb.open(options.shared.directories.directories)
+    val configDb = ConfigDb.open(options.shared.directories.directories.dbPath.toNIO)
```

Comment:
i`ConfigDb.open(options.shared.directories.directories.dbPath.toNIO)` could be extracted to `SharedOptionsUtil` to avoid repeating it in every command relying on the `configDb`

Reply:
(or `ConfigUtil`)

Reply:
Good point, should be fixed.